googleapis/google-api-nodejs-client
# docs: Fix generated documentation

Generated documentation was using the base directory & package.json for all libraries. Fixed the directories & package jsons to use.

It must have worked at one point, bc the doc sites have some classes as direct links.

I'm not sure `npm test` passes - it's timing out on my toaster.

## To test:
- On `main`, run `npm run docs`, then open one of the generated `index.html`s. You will see "Google APIs Node.js Client"
- On this branch, run `npm run docs`, then open a generated `index.html`. You should see the name of the API at the top, and the "classes" link on the left should show & work.
- Verify that the URL paths for the "Classes" are the same as the originals.

Fixes #3460 and #3405 🦕
